src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -610,36 +613,36 @@ public sealed partial class Strings
         public static int Len(short Expression) { throw null; }
         public static int Len(int Expression) { throw null; }
         public static int Len(long Expression) { throw null; }
-        public static int Len(object Expression) { throw null; }
+        public static int Len(object? Expression) { throw null; }
         [System.CLSCompliantAttribute(false)]
         public static int Len(sbyte Expression) { throw null; }
         public static int Len(float Expression) { throw null; }
-        public static int Len(string Expression) { throw null; }
+        public static int Len(string? Expression) { throw null; }
         [System.CLSCompliantAttribute(false)]
         public static int Len(ushort Expression) { throw null; }
         [System.CLSCompliantAttribute(false)]
         public static int Len(uint Expression) { throw null; }
         [System.CLSCompliantAttribute(false)]
         public static int Len(ulong Expression) { throw null; }
-        public static string LSet(string Source, int Length) { throw null; }
-        public static string LTrim(string str) { throw null; }
-        public static string Mid(string str, int Start) { throw null; }
-        public static string Mid(string str, int Start, int Length) { throw null; }
-        public static string Replace(string Expression, string Find, string Replacement, int Start = 1, int Count = -1, [Microsoft.VisualBasic.CompilerServices.OptionCompareAttribute] Microsoft.VisualBasic.CompareMethod Compare = Microsoft.VisualBasic.CompareMethod.Binary) { throw null; }
-        public static string Right(string str, int Length) { throw null; }
-        public static string RSet(string Source, int Length) { throw null; }
-        public static string RTrim(string str) { throw null; }
+        public static string LSet(string? Source, int Length) { throw null; }
+        public static string LTrim(string? str) { throw null; }
+        public static string Mid(string? str, int Start) { throw null; }
```

Comment:
>string [](start = 22, length = 6)

It looks like this 2 argument overload of `Mid` returns `string?` while the 3 argument overload return `string!`.